src/util/spicemanager.cpp

```diff
@@ -154,6 +154,7 @@ SpiceManager::SpiceManager() {
     erract_c("SET", 0, const_cast<char*>("REPORT"));
     // But we do not want SPICE to print the errors, we will fetch them ourselves
     errprt_c("SET", 0, const_cast<char*>("NONE"));
+    loadLeapSecondsSpiceKernel();
```

Comment:
```suggestion

    loadLeapSecondsSpiceKernel();
```

just to give it some visual separation to make it more obvious that it is happening